about.html

```diff
@@ -0,0 +1,189 @@
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css"
+    integrity="sha384-vSIIfh2YWi9wW0r9iZe7RJPrKwp6bG+s9QZMoITbCckVJqGCCRhc+ccxNcdpHuYu" crossorigin="anonymous">
+</head>
+
+<body>
+  <div id="main-container">
+    <!-- Top Navigations Bar -->
+    <div class="bg-top">
+      <nav class="first-nav">
+        <ul>
+          <li class="social-media-icons"><i class="fab fa-facebook-f"></i></li>
+          <li class="social-media-icons"><i class="fab fa-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </nav>
+      <nav class="second-nav">
+        <ul>
+          <li id="logo-about"><a href="index.html"><img src="./icons/micro.png" alt="logo"></a></li>
+          <li><a href="about.html">About</a></li>
+          <li>Assembler</li>
+          <li>C</li>
+          <li>JavaScript</li>
+          <li>Zsh</li>
+          <li>Interfaces</li>
+        </ul>
+      </nav>
+      <!-- Mobile Menu -->
+      <div class="menu-container">
+        <div class="menu-tab">
+          <div class="menu-tab-header">
+            <h1>Hello! Welcome</h1>
+            <button type="button" class="menu-tab-close">X</button>
+          </div>
+          <div class="menu-tab-links">
+            <h2>Home</h2>
+            <span id="arrowClose" class="material-icons"><a href="index.html">east</a></span>
+          </div>
+          <div class="menu-tab-links">
+            <h2>About</h2>
+            <span id="arrowClose2" class="material-icons"><a href="about.html">east</a></span>
+          </div>
+          <div class="menu-tab-links">
+            <h2>JavaScript</h2>
+            <span class="material-icons"><a href="#">east</a></span>
+          </div>
+          <div class="menu-tab-links">
+            <h2>Zsh</h2>
+            <span class="material-icons"><a href="#">east</a></span>
+          </div>
+          <div class="menu-tab-links">
+            <h2>Interfaces</h2>
+            <span class="material-icons"><a href="#">east</a></span>
+          </div>
+        </div>
+      </div>
+      <!-- Mobile Home page -->
+      <div id="bg-menu" class="bg-menu">
+        <button id="bg-menu-btn" type="button">
+          <span class="btn-line"></span>
+          <span class="btn-line"></span>
+          <span class="btn-line"></span>
+        </button>
+        <h2>"Make things possible"</h2>
+        <div class="home-link">
+          <ul>
+            <li><a href="./index.html"><i class="fas fa-home"></i></a>Home</li>
+            <li><a href=""><i class="fas fa-chevron-right"></i></a>About</li>
+            <li><a href=""><i class="fas fa-chevron-right"></i></a>Zsh</li>
+          </ul>
+        </div>
+      </div>
+
+      <header id="bg-header">
+        <h1>What makes technology possible?</h1>
```

Comment:
- [ ] Please fix this h1 as well, remember that a page should only have one h1 tag for the main heading.